src/main/java/seedu/address/model/AddressBook.java

```diff
@@ -158,6 +158,13 @@ public boolean removePerson(ReadOnlyPerson key) throws PersonNotFoundException {
         }
     }
 
+    /**
+     * Sorts the persons object alphanumerically by name.
+     */
+    public void sortData() {
```

Comment:
Consider if this method should be private.

Reply:
Cant cos its being called in another class